datafiles/templates/UserSignupReset/SignupRequest.html.st

```diff
@@ -48,4 +56,27 @@ can set your password and activate your account.
 </form>
 
 </div>
+<script type="text/javascript">
+var image = document.getElementById("image");
+var timestamp = document.getElementById("timestamp");
+var hash = document.getElementById("hash");
+
+function changeCaptcha() {
+  var xmlHttp = new XMLHttpRequest();
```

Comment:
Maybe use [fetch](https://caniuse.com/fetch) ?

Reply:
Sorry for being busy these days. I'm not familiar with javascript; is there a reason for preferring fetch to XMLHttpRequest?

Reply:
Generally it is [easier to use](https://medium.com/beginners-guide-to-mobile-web-development/the-fetch-api-2c962591f5c) than `XMLHttpRequest`.
It doesn't support IE, but who cares IE. :D